conf/messages.en

```diff
@@ -64,6 +64,9 @@ viewArrivalNotifications.table.trader.name = Trader name
 viewArrivalNotifications.table.office      = Office
 viewArrivalNotifications.table.procedure   = Procedure
 viewArrivalNotifications.table.status      = Status
+viewArrivalNotifications.table.status.unloadingPermission = You can unload
+viewArrivalNotifications.table.status.arrivalSubmitted = Arrival notification sent
+unloadingPermission.link.title = Make unloading remarks
```

Comment:
Consider renaming these to keep consistent

Reply:
changed to
 movement.status.unloadingPermsission
 movement.status.arrivalSubmitted